services/node/node-current.tester.js

```diff
@@ -13,98 +13,164 @@ function expectSemverRange(message) {
   expect(() => new Range(message)).not.to.throw()
 }
 
-t.create('engines satisfies current node version')
+t.create('gets the node version of passport - live')
```

Comment:
you can drop the ` - live` suffix from the test name. our test runner actually detects whether tests are live or mocked and annotates them accordingly

Reply:
will do